scripts/utils.ts

```diff
@@ -36,3 +40,87 @@ export async function downloadLatestApis(
     throw err;
   }
 }
+
+/**
+ * Downloads API assets using anypoint-cli-v4 and extracts them to the target directory.
+ *
+ * @param apiId - The API ID in Anypoint Exchange
+ * @param targetDir - Directory where the API assets should be extracted
+ * @param options - Optional parameters
+ * @param options.anypointCliPath - Path to anypoint-cli-v4 executable (default: 'anypoint-cli-v4')
+ * @param options.extract - Whether to extract the downloaded assets (default: true)
+ * @param options.overwrite - Whether to overwrite existing files (default: true)
+ *
+ * @returns Promise that resolves when the download and extraction is complete
+ */
+export async function downloadApisWithAnypointCli(
```

Comment:
Oh, that's cool that anypoint exchange has a CLI tool wrapped as an npm package, I don't think this was available when `raml-toolkit` was created

Should this download logic be tucked into `raml-toolkit` and then exposed as a helper function for the isomorphic SDK to consume? That's the current pattern, where `raml-toolkit` holds the logic/tooling for downloading and SDK generation and the SDK holds the templates and APIs. I'm okay with either approach